### PR TITLE
[ADD] stock_picking_sms

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

**Review by mgosai — APPROVED** [review]
LGTM

File: osi_panda_sms_delivery_notification/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        for picking in self:
+            note_msg = _('SMS text message reminder sent !')
+            if picking.picking_type_id.code == 'internal':
+                msg = "We are glad to inform you that your order " \
+                      "n° %s has been shipped." % (picking.origin)
+                picking.message_post_send_sms(msg, note_msg=note_msg)
+            elif picking.picking_type_id.code == 'outgoing':
+                msg = "We are glad to inform you that your order " \
+                      "n° %s has been delivered." % (picking.origin)
+                picking.message_post_send_sms(msg, note_msg=note_msg)
```
Comment (max3903) [inline osi_panda_sms_delivery_notification/models/stock_picking.py]:
```suggestion
            if picking.picking_type_id.code == 'internal':
                msg = "We are glad to inform you that your order " \
                      "n° %s has been shipped." % (picking.origin)
            elif picking.picking_type_id.code == 'outgoing':
                msg = "We are glad to inform you that your order " \
                      "n° %s has been delivered." % (picking.origin)
            picking.message_post_send_sms(msg, note_msg=note_msg)
```

File: osi_panda_sms_delivery_notification/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        pickings = super(StockPicking, self).action_done()
+        for picking in self:
+            note_msg = _('SMS text message reminder sent !')
+            if picking.picking_type_id.code == 'internal':
+                msg = "We are glad to inform you that your order " \
+                      "n° %s has been shipped." % (picking.origin)
+            elif picking.picking_type_id.code == 'outgoing':
+                msg = "We are glad to inform you that your order " \
+                      "n° %s has been delivered." % (picking.origin)
+            picking.message_post_send_sms(msg, note_msg=note_msg)
```
Comment (max3903) [inline osi_panda_sms_delivery_notification/models/stock_picking.py]:
```suggestion
            if msg:
                picking.message_post_send_sms(msg, note_msg=note_msg)
```

**Review by max3903 — APPROVED** [review]

File: osi_panda_sms_delivery_notification/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.multi
+    def action_done(self):
+        pickings = super(StockPicking, self).action_done()
+        for picking in self:
+            note_msg = _('SMS text message reminder sent !')
```
Comment (bodedra) [inline osi_panda_sms_delivery_notification/models/stock_picking.py]:
```suggestion
            note_msg = _('SMS text message reminder sent !')
            msg = ""
```

**Review by bodedra — COMMENTED** [review]
It will throw error for incoming goods.

File: osi_panda_sms_delivery_notification/models/stock_picking.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            msg = ""
```
Comment (max3903) [inline osi_panda_sms_delivery_notification/models/stock_picking.py]:
```suggestion
            msg = False
```

**Pull request merged**